cdc/owner.go

```diff
@@ -529,6 +539,33 @@ loop:
 	return nil
 }
 
+func (o *Owner) watchFeedChange(ctx context.Context) chan struct{} {
+	output := make(chan struct{}, 1)
+	go func() {
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			default:
+			}
+			wch := o.etcdClient.Client.Watch(ctx, kv.TaskPositionKeyPrefix, clientv3.WithFilterDelete(), clientv3.WithPrefix())
+
+			for resp := range wch {
+				if resp.Err() != nil {
```

Comment:
Do we need to process etcd `ErrCompact` here?

Reply:
Not necessary. Any kind of error will simply recreate the watch channel, and it will terminated when `ctx` is canceled.